src/libs/ajax.js

```diff
@@ -150,8 +109,6 @@ const withRequesterPays = wrappedFetch => (url, ...args) => {
   return tryRequest()
 }
 
-export const fetchOk = _.flow(withInstrumentation, withCancellation, withErrorRejection)(fetch)
-
 const fetchSam = _.flow(withUrlPrefix(`${getConfig().samUrlRoot}/`), withAppIdentifier)(fetchOk)
```

Comment:
I'd vote we go ahead and move the other `with...` wrappers and `fetch...` functions to ajax-common.js as well.

Reply:
Sure. I tend to be conservative with moving things... this will just make it easier for others to split their stuff out though so Im for it.